lmp/bb-build.sh

```diff
@@ -21,38 +52,13 @@ bitbake -e ${IMAGE} > ${archive}/bitbake_image_env.txt
 status "Run bitbake (setscene tasks only)"
 bitbake --setscene-only ${IMAGE} || true
 
+# add trap to do some pending operations on exit
+trap finish TERM INT EXIT
```

Comment:
And at this commit please add a statement saying that this shouldn't imply in any functional changes, it is just moving code around and making it being called on trap.

Reply:
we need to return the same exit code as before, also added now inside the trap